src/Microsoft.ML.Featurizers/DateTimeTransformer.cs

```diff
@@ -667,74 +681,68 @@ protected override DataViewSchema.DetachedColumn[] GetOutputColumnsCore()
                 return columns.ToArray();
             }
 
-            private Delegate MakeGetter<T>(DataViewRow input, int iinfo)
+            private Delegate MakeGetter<TInput, TTransformed>(DataViewRow input, int iinfo)
             {
-                var getter = input.GetGetter<long>(input.Schema[_parent._column.Source]);
-                ValueGetter<T> result = (ref T dst) =>
+                var getter = input.GetGetter<TInput>(input.Schema[_parent._column.Source]);
+
+                ValueGetter<TTransformed> result = (ref TTransformed dst) =>
                 {
-                    long dateTime = default;
-                    getter(ref dateTime);
+                    TInput dateTimeColumn = default;
+                    getter(ref dateTimeColumn);
 
-                    if (!_cache.TryGetValue(dateTime, out TimePoint timePoint))
```

Comment:
I am curious about this deleted code. What was the cache for? Why is it no longer needed?

Reply:
I was trying to optimize by caching the retrieved value. Since this code can run in parallel though, the cache was thread safe. After doing some testing during this change it turned out in this case the thread safe cache was slower then just calling the native code extra times.